src/Service.GraphQLBuilder/GraphQLNaming.cs

```diff
@@ -28,6 +28,8 @@ public static class GraphQLNaming
         /// <seealso cref="https://spec.graphql.org/October2021/#sec-Names.Reserved-Names"/>
         public const string INTROSPECTION_FIELD_PREFIX = "__";
 
+        public const string LINKING_OBJECT_PREFIX = "linkingObject";
```

Comment:
nit: Should this be `linkingObject_` because we for linking entity names, the pattern `LinkingEntity_{entity1}{entity2}` is followed?

Reply:
The pattern for linking entity is `LinkingEntity$entity1$entity2`. Also since the GQL linking object names are exposed, not keeping `_` to stay consistent with the current naming convention we follow.